Automattic/vip-go-mu-plugins
# Search: fix a typo resulting in 'Unknown error type' in logs

## Description

`$response_error` was not set, the correct variable is `$response['error']`. This PR fixes that.

## Changelog Description

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Apply the PR
2. Trigger some errors and verify the correct error type and root cause are set.
